src/pages/management/webhooks/index.mdx

```diff
@@ -15,7 +15,7 @@ In orderd to receive webhooks, you need to [register](/management/configuration-
 
 - [See webhook methods.](/management/configuration-api/v3.4/#webhooks)
 - [Learn how to use webhooks with the LiveChat API.](https://developers.livechat.com/updates/setting-up-webhooks-with-livechat-api/)
-- [Read the Changelog.](../changelog/)
+- [Read the Changelog.](/management/changelog/)
```

Comment:
I fixed it that way (instead of just .) as perhaps in the future we'll be moving things around so we can easily find this link again

Reply:
I reverted this change as I saw that Kama has a task for it :D